app/Http/Controllers/RequestController.php

```diff
@@ -108,4 +108,32 @@ public function changeRequestStatus($id)
             ]);
         }
     }
+
+    public function getRequestsByStatus($id, Request $request)
+    {
+        try {
+            $data = $this->requestRepository->find($id);
+            if ($data->status == $request->newStatus) {
+                return response()->json([
+                    'results' => $data,
+                    'success' => true,
+                    'message' => 'Get Request successfully',
+                ]);
```

Comment:
find($id) để làm gì nhỉ ?

Reply:
tôi truyền cho nó cái id của request rồi nhập status vào. nếu đúng thì trả về request đấy

Reply:
vậy tức là tìm được 1 thằng request rồi. nếu status của nó không phải == $request->status thì nó sẽ không trả về thằng nào hả ?

Reply:
uk nhỉ, có vấn đề thật. thế thì cần gì get status nữa, get by id là lấy đc request đấy rồi nhỉ

Reply:
@nhoconline xem lại logic nhé. Chỉ cần truyền vào status rồi lấy ra những record có status = status đấy thôi.

Reply:
nghĩa là trong bảng request thằng nào có status là "Not Resolve" thì hiển thị thông tin những thằng đấy ra à